gradle/dependencies.gradle

```diff
@@ -22,29 +22,41 @@ ext {
   libs = [:]
   
   // Enabled by default when commands like `testAll` are invoked
-  defaultScalaVersions = [ '2.11', '2.12' ]
+  defaultScalaVersions = [ '2.11', '2.12', '2.13' ]
   // Available if -PscalaVersion is used. This is useful when we want to support a Scala version that has
   // a higher minimum Java requirement than Kafka. This was previously the case for Scala 2.12 and Java 7.
-  availableScalaVersions = [ '2.11', '2.12' ]
+  availableScalaVersions = [ '2.11', '2.12', '2.13' ]
 }
 
 // Add Scala version
 def defaultScala211Version = '2.11.12'
 def defaultScala212Version = '2.12.8'
+def defaultScala213Version = '2.13.0'
 if (hasProperty('scalaVersion')) {
   if (scalaVersion == '2.11') {
     versions["scala"] = defaultScala211Version
   } else if (scalaVersion == '2.12') {
     versions["scala"] = defaultScala212Version
-  } else {
+  } else if (scalaVersion == '2.13') {
+    versions["scala"] = defaultScala213Version
+  }  else {
     versions["scala"] = scalaVersion
   }
 } else {
   versions["scala"] = defaultScala211Version
```

Comment:
This should be `defaultScala212Version` since we default to 2.12 now. An existing issue, but maybe we can fix it at the same time.

Reply:
Done, commit included/squashed.